hack/pman/pman.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash -e
```

Comment:
This one definitely is too complex to be written in `bash`. It needs proper testing.

Reply:
see please https://github.com/kubevirt/kubevirtci/pull/957#discussion_r1130801292

Reply:
My experience tells me that the move to a proper language will never happen.

Reply:
BTW, why add Python when go is already available>

Reply:
I already have an action item to move to python so it will happen
I prefer to do this tool in its own repo as python, it is more suitable, and has very nice pytest
Here we will just consume it (as it will be a container image)
this way more projects might able to use it

Reply:
Another option would be that we integrate it into the existing gocli which is present already. There would be no overhead. Also I can't review Python code, so I would be against introducing it here if the need is not there.

Reply:
We have people that know to review it, and we can all learn it
I am learning it on the fly, and it is an amazing language.

I think we should have a broader forum to decide please, as I wasn't part of the decision even.
Can we do a meeting about it ? wdyt ?